externals/llvm-external-projects/torch-mlir-dialects/test/Dialect/Tcp/dummy_op.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: torch-mlir-dialects-opt %s | torch-mlir-dialects-opt | FileCheck %s
```

Comment:
nit: file name should be dummy-op.mlir for consistency.

Reply:
Done